src/maud/stan_code/inference_model_lower_blocks.stan

```diff
@@ -54,18 +54,20 @@ transformed data {
 parameters {
   vector[N_metabolite] formation_energy;
   vector<lower=0>[N_kinetic_parameters] kinetic_parameters;
-  vector<lower=0>[N_enzyme] enzyme_concentration[N_experiment];
-  vector<lower=0>[N_unbalanced] conc_unbalanced[N_experiment];
+  matrix<lower=0>[N_experiment, N_enzyme] enzyme_concentration;
+  matrix<lower=0>[N_experiment, N_unbalanced] conc_unbalanced;
 }
 transformed parameters {
   real initial_time = 0;
   vector<lower=0>[N_mic] conc[N_experiment];
-  vector[N_reaction] flux[N_experiment];
+  matrix[N_experiment, N_reaction] flux;
   vector[N_enzyme] delta_g = stoichiometric_matrix' * formation_energy[metabolite_ix_stoichiometric_matrix];
   for (e in 1:N_experiment){
+    vector[N_enzyme] temp_enzyme_concentration = enzyme_concentration[e, ]';
+    vector[N_enzyme] temp_unbalanced_concentration = conc_unbalanced[e, ]';
```

Comment:
I think avoiding introducing the extra variables would make the code a bit easier to read (i.e. instead of declaring `temp_enzyme_concentration` and `temp_unbalanced_concentration` just refer to `enzyme_concentration[e, ]'` later)